source/extensions/omni.isaac.lab/omni/isaac/lab/terrains/terrain_importer_cfg.py

```diff
@@ -36,7 +36,7 @@ class TerrainImporterCfg:
     All sub-terrains are imported relative to this prim path.
     """
 
-    num_envs: int = MISSING
+    num_envs: int = 1
     """The number of environment origins to consider."""
```

Comment:
```suggestion
    """The number of environment origins to consider. Defaults to 1.
    
    In case, the :class:`~omni.isaac.lab.scene.InteractiveSceneCfg` is used, this parameter gets overidden by
    :class:`~omni.isaac.lab.scene.InteractiveSceneCfg.num_envs` attribute.
    """
```